src/hotspot/cpu/riscv/riscv_v.ad

```diff
@@ -3784,6 +3787,37 @@ instruct vconvF2HF(vReg dst, vReg src, vReg vtmp, vRegMask_V0 v0, iRegINoSp tmp)
   ins_pipe(pipe_slow);
 %}
 
+
+// ------------------------------ Popcount vector ------------------------------
+
+instruct vpopcount_mask(vReg dst_src, vRegMask_V0 v0) %{
+  match(Set dst_src (PopCountVI dst_src v0));
+  match(Set dst_src (PopCountVL dst_src v0));
```

Comment:
Is there a reason to force input & output being the same vector register?

Reply:
Seems not, I modify it to usual pattern.